com.wings3d.WINGS.json

```diff
@@ -173,30 +181,30 @@
       "name": "wings",
       "buildsystem": "simple",
       "build-options": {
-	"no-debuginfo": true,
-	"env": {
-	  "GL_LIBS": "-L/app/lib",
-	  "HOME": "/app/lib"
-	}
+        "no-debuginfo": true,
+        "env": {
+          "GL_LIBS": "-L/app/lib",
+          "HOME": "/app/lib"
+        }
       },
       "build-commands": [
         "mv ${FLATPAK_DEST}/_deps .",
-	"make unix",
-	"./wings-*-linux.bzip2.run",
-	"install -d /app/share/applications /app/share/appdata",
-	"sed -i 's/^Exec=.*/Exec=wings %f/' /app/lib/wings-*/wings.desktop",
-	"install /app/lib/wings-*/wings.desktop /app/share/applications/com.wings3d.WINGS.desktop",
-	"install /app/lib/wings-*/wings.appdata.xml /app/share/appdata/com.wings3d.WINGS.appdata.xml",
-	"install -D /app/lib/wings-*/lib/wings*/ebin/wings_icon_48x48.png /app/share/icons/hicolor/48x48/apps/wings.png",
-	"install -D /app/lib/wings-*/lib/wings*/ebin/wings_icon_256x256.png /app/share/icons/hicolor/256x256/apps/wings.png",
-	"install /app/lib/wings-*/wings /app/bin"
+        "make unix",
+        "./wings-*-linux.bzip2.run",
+        "install -d /app/share/applications /app/share/appdata",
+        "sed -i 's/^Exec=.*/Exec=wings %f/' /app/lib/wings-*/wings.desktop",
+        "install /app/lib/wings-*/wings.desktop /app/share/applications/com.wings3d.WINGS.desktop",
+        "install /app/lib/wings-*/wings.appdata.xml /app/share/appdata/com.wings3d.WINGS.appdata.xml",
+        "install -D /app/lib/wings-*/lib/wings*/ebin/wings_icon_48x48.png /app/share/icons/hicolor/48x48/apps/wings.png",
+        "install -D /app/lib/wings-*/lib/wings*/ebin/wings_icon_256x256.png /app/share/icons/hicolor/256x256/apps/wings.png",
+        "install /app/lib/wings-*/wings /app/bin"
       ],
       "sources": [
         {
-	  "type": "git",
-	  "url": "https://github.com/dgud/wings.git",
-	  "tag": "v2.2.7",
-	  "commit": "a64f8f766cef0c197dc23d511764103d65ab376c"
+          "type": "git",
+          "url": "https://github.com/dgud/wings.git",
+	  /* "tag": "v2.3", */
```

Comment:
The tag should be ok

Reply:
> The tag should be ok

In this case we remove the commit?

Reply:
I think you need both, but don't remember.